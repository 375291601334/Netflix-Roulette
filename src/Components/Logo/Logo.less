@import '../../Styles/variables.less';

.logo {
  display: inline-block;
  font-size: 20px;
  color: @red;
  
  span {
    font-family: @MontserratBoldFont;
  }
}
