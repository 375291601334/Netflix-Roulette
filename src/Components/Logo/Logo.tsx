import React from 'react';
import css from './Logo.less';

export function Logo() {
  return <div className={css.logo}><span>netflix</span>roulette</div>;
}
