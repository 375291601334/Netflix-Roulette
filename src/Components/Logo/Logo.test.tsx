import React from 'react';
import { render } from '@testing-library/react';
import { Logo } from './Logo';

describe('Logo', () => {
  test('render Logo', () => {
    const { baseElement } = render(<Logo />);
    expect(baseElement).toMatchSnapshot();
  });
});
