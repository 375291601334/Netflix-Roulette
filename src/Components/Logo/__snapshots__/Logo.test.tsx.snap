// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Logo render Logo 1`] = `
<body>
  <div>
    <div
      class="logo undefined"
    >
      <span>
        netflix
      </span>
      roulette
    </div>
  </div>
</body>
`;
